src/index.js

```diff
@@ -746,6 +810,9 @@ export const fuzzExpression = (f = new FuzzerState, {allowIdentifierExpression =
   if (f.allowYieldExpr) {
     fuzzers = fuzzers.concat(yieldExprFuzzers);
   }
+  if (f.allowAwaitExpr) {
+    fuzzers = fuzzers.concat(fuzzAwaitExpression);
```

Comment:
```suggestion
    fuzzers = fuzzers.concat([fuzzAwaitExpression]);
```

Right?

Reply:
Also, are we trying to avoid mutation? We could just `push`.

Reply:
We are trying to avoid mutation, yes. We could clone then push, but whatever.